stl/inc/chrono

```diff
@@ -2176,7 +2176,8 @@ namespace chrono {
 
     _NODISCARD inline string _Tzdb_update_version(const string_view _Version, const size_t _Num_leap_seconds) {
         string _Icu_version{_Version.substr(0, _Version.find_last_of('.'))};
-        return _STD move(_Icu_version) + "." + _STD to_string(_Num_leap_seconds);
+        (_Icu_version += '.') += _STD to_string(_Num_leap_seconds);

```

Comment:
We conventionally avoid complicated subexpressions (especially in newer code). Spending editor lines to improve clarity is a worthwhile cost. Here, we should perform the `+=` concatenations on separate lines.